contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -3004,6 +3004,15 @@ SELECT
 WHERE FALSE; -- This condition will ensure that the view is empty
 GRANT SELECT ON sys.numbered_procedures TO PUBLIC;
 
+ALTER FUNCTION sys.fn_mapped_system_error_list() RENAME TO fn_mapped_system_error_list_deprecated_in_2_2_0;
```

Comment:
this will cause existing users run into error as underlying C function has changed. We should validate the behaviour via upgrade tests. Suprio has comment regarding existing app being broken. We have both option to break them or keep working.

Reply:
Going with an option of not breaking the existing application.